wallet_transactions/wallets/views.py

```diff
@@ -0,0 +1,72 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework.reverse import reverse
+from rest_framework import status
+
+from .models import Wallet, COUNTOFWALLETYOUCANCREAT
+from .serializers import WalletSerializer
+
+import random
+import string
+
+CHECK = True
+
+
+@api_view(("GET",))
+def wallet_list(request):
+    """Create view list for all Wallet"""
+    wallet = Wallet.objects.filter(owner__username=request.user)
+    serializer = WalletSerializer(wallet, many=True)
+    return Response(serializer.data)
+
+
+@api_view(["POST"])
+def wallet_create(request):
+    """Create view creale for Wallet"""
+    global CHECK, name
+    serializer = WalletSerializer(data=request.data)
+    user = request.user
+    while CHECK:
+        name = [random.choice(string.ascii_uppercase + string.digits) for i in range(8)]
+        check_name = Wallet.objects.filter(name=name).count()
+        if check_name == 0:
+            CHECK = False
```

Comment:
it's better to add the logic with name generation to the wallet's model

Reply:
I moved this logic to wallet's model